exercises/practice/collatz-conjecture/.meta/example.cairo

```diff
@@ -1,6 +1,12 @@
-pub fn steps(number: u64) -> u64 {
+pub fn steps(number: usize) -> Option<usize> {
+    if number == 0 {
```

Comment:
```suggestion
    assert!(number != 0, "Only positive integers are allowed");
```
Follows the [canonical data](https://github.com/exercism/problem-specifications/blob/main/exercises/collatz-conjecture/canonical-data.json#L48) more closely, which expects an actual error. This also removes the need for `Option`.

To test for this, you can use the `should_panic` attribute:
https://book.cairo-lang.org/ch10-01-how-to-write-tests.html#checking-for-panics-with-should_panic